lib/tools/node-modules/node-modules-dest-copy.ts

```diff
@@ -46,8 +46,8 @@ export class TnsModulesCopy {
 			shelljs.mkdir("-p", targetDir);
 			shelljs.cp("-Rf", dependency.directory, targetDir);
 
-			//remove platform-specific files (processed separately by plugin services)
-			const targetPackageDir = path.join(targetDir, dependency.name);
+			// remove platform-specific files (processed separately by plugin services)
+			const targetPackageDir = path.join(isScoped ? this.outputRoot : targetDir, dependency.name);
```

Comment:
Is this really correct? If I understand correctly, the idea here is to copy `<project dir>/node_modules/@<scoped_name>/<package_name>` to `<projectDir>/platforms/<path to tns_modules>/@<scoped_name>/<package_name>` and remove the platforms dir from the destination. From this code it looks like we'll try to delete `<projectDir>/platforms/<path to tns_modules>/platforms` dir. Am I correct?